hera_cal/abscal.py

```diff
@@ -4004,6 +4004,7 @@ def post_redcal_abscal_argparser():
     a.add_argument("--phs_conv_crit", default=1e-6, type=float, help="convergence criterion for updates to iterative phase calibration that compares them to all 1.0s.")
     a.add_argument("--clobber", default=False, action="store_true", help="overwrites existing abscal calfits file at the output path")
     a.add_argument("--verbose", default=False, action="store_true", help="print calibration progress updates")
+    a.add_argument("--use_abs_amp_lincal", defualt=True, action="store_false", help="finish calibration with an unbiased amplitude lincal step")
```

Comment:
I'd call this `"--skip_abs_amp_lincal"` with default False and action store_true. Since the default behavior is to use abs_amp_lincal, you only want this commandline argument to be used when you want to do something other than the default. You'll also have to fix it in the run script too.